U/UCX/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "UCX"
+version = v"1.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/openucx/ucx/releases/download/v1.7.0/ucx-1.7.0.tar.gz" =>
+    "6ab81ee187bfd554fe7e549da93a11bfac420df87d99ee61ffab7bb19bdd3371",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd ucx-*
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    --disable-numa
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
```

Comment:
According to the [`README`](https://github.com/openucx/ucx/blob/93540351a13b691fad620ad2f80abb9587ed8134/README#L178-L182) it should work also for aarch64?

Reply:
Yeah I didn't spend to much time investigating since I wanted to get some of the dependencies first. https://github.com/JuliaPackaging/Yggdrasil/pull/440